### PR TITLE
Design new prototype UI (V0).

### PR DESCRIPTION
**Pull request by juanuribe28**
The UI is update to provide a better and more logical experience. Instead of dividing the main page by keywords, news and actions, it is now divided by the different hardcoded keywords. The java servlets have been updated to work with this new design. This PR contains all the changes included in the prototype of the app (V0).

### CODE REVIEW

File: src/main/webapp/json/news.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "articles": [
-    {
+  "articles": {
+    "Black Lives Matter" : [{
```
Comment (nssafir) [inline src/main/webapp/json/news.json:3]: Is the idea that there are multiple articles for each keyword?

File: src/main/webapp/main.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (nssafir) [inline src/main/webapp/main.html:3]: [Whole file] some of the indentation in this file is off.

File: src/main/webapp/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+body.focus {
+  background: linear-gradient(var(--upperColor) 0, 
+    var(--upperColor) 49.5vh, 
+    var(--lowerColor) 50.5vh, 
+    var(--lowerColor));
+  grid: [row1-start] "content " 100vh [row1-end]
+        / 1fr;
+	place-items: strech strech;
```
Comment (nssafir) [inline src/main/webapp/styles/main.css]: [Nit] indentation seems off here.

File: src/main/webapp/styles/main.css [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Open Sans', sans-serif;;
+}
+
+#content {
+	width: 100vw;
```
Comment (nssafir) [inline src/main/webapp/styles/main.css]: [Nit] see above

File: src/main/webapp/styles/main.css [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+  height: 100%;
+  grid-area: content;
+}
+
+#keywords {
+	margin: auto;
```
Comment (nssafir) [inline src/main/webapp/styles/main.css]: [Nit] are you using four spaces or two spaces for indentations?

File: src/main/webapp/templates/keywords.html [hunk 6]
```diff
@@ -1,15 +1,37 @@
...
-  <input type="submit" />
-</form>
-<div id="keyword">
-  <ul>
-    {{#keywords}}
-    <li>{{term}}</li>
-    {{/keywords}}
-  </ul>
+<div id="keywords"> 
+  {{#keyterms}}
```
Comment (dranilew) [inline src/main/webapp/templates/keywords.html]: This should be looping through "keywords", not "keyterms" (I think in general we should keep it as "keywords")

File: src/main/webapp/main.html (hunk 2 above)
Comment (juanuribe28) [reply src/main/webapp/main.html:3]: Yeah, there was a problem when I moved the files from codepen to cloudshell, but I fix it now.

File: src/main/webapp/json/news.json (hunk 1 above)
Comment (juanuribe28) [reply src/main/webapp/json/news.json:3]: Yeah

**Review by dranilew — COMMENTED** [review]
I'm not sure if it's more because of GitHub or something, but the formatting in some of the files were really weird (probably due to using a tab character instead of manually putting in two spaces), especially in the CSS files. I've marked pretty much all of them, I think, but everything else looks good to me.

EDIT: You seem to have fixed it all by yourself now, this comment is void.

**Review by nssafir — APPROVED** [review]

**Review by seangilpin — APPROVED** [review]

**Pull request merged**